cms/templates/partials/enterprise/learning_journey.html

```diff
@@ -0,0 +1,24 @@
+{% load static image_version_url wagtailcore_tags %}
+
+<div class="learning-journey-block">
+    <div class="container">
+        <h1>{{ page.heading|upper|richtext }}</h1>
+        <div class="learning-journey-container">
+            <div class="learning-journey-content">
+                <ul>
+                    {% for journey_item in learning_journey.journey_items %}
+                        <li>{{ journey_item }}</li>
+                    {% endfor %}
+                </ul>
+                <p>{{ page.description }}</p>
```

Comment:
```suggestion
                <p>{{ page.description|richtext }}</p>
```
<img width="905" alt="Screen Shot 2024-01-08 at 3 05 18 PM" src="https://github.com/mitodl/mitxpro/assets/52656433/5ddd7316-2188-46cc-a52e-5fbe22e4bcc1">